app/src/test/java/at/aau/edu/wizards/gameModel/GameModelListenerUnitTest.kt

```diff
@@ -7,6 +7,7 @@ import org.junit.jupiter.api.Test
 class GameModelListenerUnitTest {
     private val viewModel = null
 
+    // FIXME too large test
```

Comment:
This is fair - tests are really not that well written. I'm aware this is not good practice but I don't mind taking the L in terms of grading for it. Even then, I'd rather expand tests than put in the effort to split these.

Reply:
done